backend/src/test/java/botobo/core/documentation/UserDocumentationTest.java

```diff
@@ -82,4 +85,65 @@ void updateProfile() throws Exception {
                 .status(status().isOk())
                 .identifier("users-update-profile-get-success");
     }
+
+    @Test
+    @DisplayName("회원 정보 수정 - 성공")
+    void update() throws Exception {
+        String token = "botobo.access.token";
+        UserUpdateRequest userUpdateRequest = UserUpdateRequest.builder()
+                .userName("수정된_이름")
+                .bio("수정된 바이오")
+                .profileUrl("profile.io")
+                .build();
+
+        UserResponse userResponse = UserResponse.builder()
+                .id(1L)
+                .userName("이름")
+                .profileUrl("profile.io")
+                .build();
+
+        AppUser appUser = AppUser.builder()
+                .id(1L)
+                .role(Role.USER)
+                .build();
+
+        given(authService.findAppUserByToken(token)).willReturn(appUser);
+        given(userService.update(1L, userUpdateRequest, appUser)).willReturn(userResponse);
```

Comment:
현재 문서화 작업이 잘 안되고 있는 상황이에요. 응답이 안 보여요!
임시로 아래처럼 처리해 주실 수 있나요? 리졸버 관련하여 모킹이 제대로 안 되어있어서 생기는 문젠데,
제가 이전 좋아요 PR에서 문제집 문서화 테스트 수정할 때 주제 관련해서 수정했어서, 문서화 테스트 리팩토링할 때 다른 문서화 테스트에도 일괄 반영할게요!
```suggestion
given(userService.update(anyLong(), any(), any())).willReturn(userResponse);
```

Reply:
아하 넵넵! 우선 남겨주신 대로 처리할게요!!